README.md

```diff
@@ -39,24 +42,25 @@ AwsSsmEnv.load!(begins_with: "myapp.#{ENV['RACK_ENV']}.")
 
 ## Quick Start
 
-事前にAWS EC2 Parameter Storeにパラメータを登録しておく必要があります。
+It is necessary to register the parameters in `AWS EC2 Parameter Store` in advance.
```

Comment:
I think it's better to use `First of all,` instead of `in advance` like below:

> First of all, register the parameters in `AWS EC2 Parameter Store`.